### ISSUE TITLE
Implement possibility of "ticking" activity in advance

### ISSUE THREAD

**Issue by giovi321** [iSoron/uhabits#180]
I use the habits for many different uses. Some of the activities can be done in advance for the next 5/7 days, therefore, it would be very useful if I could "tick" in advance the activities for n days because I did it in advance.

**Comment by iSoron** [comment]
Could you give me a more specific scenario where this feature would be important? If you specify that your habit needs to be repeated only one time every 5 or 7 days, then the app will automatically add checkmarks for you (in the future).

**Comment by giovi321** [comment]
Of course. The scenario involves paying a congestion charge for accessing the city center. It has to the paid every day that you access the city center. You can also pay it in advance, so when I pay 5-7 days in advance, it would be nice to already have the checkmarks.I can think of another scenario, maybe more common. It involves preparing food in advance for the week. Let's say you cook and freeze food for the next 5 days, you should then select the tickmarks in advance.                                                                                                                                                                                                                  From:notifications@github.comSent:24 September 2016 10:58 pmTo:uhabits@noreply.github.comReply to:reply@reply.github.comCc:write@giovi.cr; author@noreply.github.comSubject:Re: [iSoron/uhabits] Implement possibility of "ticking" activity in advance (#180)  Could you give me a more specific scenario where this feature would be important? If you specify that your habit needs to be repeated only one time every 5 or 7 days, then the app will automatically add checkmarks for you (in the future).

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
